### ISSUE TITLE
rfft and performances

### ISSUE THREAD

**Issue by barakmam** [csbhr/Deep-Blind-VSR#7]
I tested your code with --quick_test Gaussian_Vid4 and got different results:

The rfft (& irfft) function is deprecated and the torch version is very old, not compatible with my python and CUDA versions. I replaced that with:
def rfft(t):
# Real-to-complex Discrete Fourier Transform, onesided=False
t = torch.fft.fft2(t)
return torch.stack([t.real, t.imag], dim=-1)

def irfft(t):
# Complex-to-real Inverse Discrete Fourier Transform, onesided=False
t = t[..., 0] + 1j*t[..., 1]
return torch.fft.ifft2(t).real

and validate that with Colab (which supports torch 0.4.1 as you used).
It seems to be accurate part to numeric errors.

The score I get:
Total AVG-PSNR=23.777, AVG-SSIM=0.7013
![image](https://user-images.githubusercontent.com/57372254/166219796-dcf4d4ff-8f2c-48e3-b6ad-7e037ab0dea1.png)

What can cause the different performance?
Thx!
